spyder_line_profiler/__init__.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2022, Spyder Line Profiler contributors
 #
-# Copyright © 2013 Spyder Project Contributors
-# Licensed under the terms of the MIT License
-# (see LICENSE.txt for details)
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+"""
+Spyder Line Profiler.
+"""
+
+__version__ = "0.5.1.dev0"
```

Comment:
I think the next release will be 0.3.0 so this should still be 0.3.0.dev0 

However, maybe I missing something, is there some reason to jump to 0.5?

Reply:
I thought it might make sense to reflect the compatibility jump from Spyder 3 to Spyder 5, by having the minor match the python major. At least I'd suggest that. But it doesn't matter much.

Reply:
Oh I see I think using 0.3.0 should be enought although maybe following the idea of a way to reflect the compatibility jump we should directly jump to 1.0.0? What do you think @ccordoba12 @jitseniesen ?

Reply:
@skjerns lets leave this as `0.3.0.dev0`

Reply:
Seems unchanged here, maybe a missing file save before doing the commit @skjerns ?

Reply:
Ah, yes, commited from the wrong folder